tableauserverclient/server/endpoint/flowruns_endpoint.py

```diff
@@ -0,0 +1,49 @@
+from tableauserverclient.models import flowrun_item
+from .endpoint import Endpoint, QuerysetEndpoint, api
+from .. import FlowRunItem, PaginationItem
+
+import logging
+
+logger = logging.getLogger("tableau.endpoint.flowruns")
+
+
+class FlowRuns(QuerysetEndpoint):
```

Comment:
I know we've already merged some endpoints in as Queryset endpoints, but I've been skeptical of the approach -- I worry about needing more and more combinations of endpoint types, rather than just relying on helper methods or wrappers (like `Pager`).

The ship has sailed, but we should probably decide what route we want to go and convert all the endpoints at once

Reply:
The `Queryset` methods I think are a nice convenience functionality for building up `RequestOptions` in a way that is a little more intuitive and fluent. I set it as the superclass for `FlowRuns` since I was working under the presumption that usually an individual querying the site will care more about getting runs for a specific flow and in a specific order than just generally querying all flows.